### ISSUE TITLE
Looking for Noble Kennamer from ICML poster session 2018

### ISSUE THREAD

**Issue by gosha1128** [NobleKennamer/ContextNet#1]
It doesn't seem right using a github issue to track you down, but it looks like your email contact from a paper is out of date.

We talked at length at your poster session at ICML recently.

I was wondering if you could help me track down some relevant research / references for my upcoming talk "AI For Mars" at NextAI, where I talk about my company's radiation hardened computational memory chip - with possible use for space satellites and exploration.

Basically, I'm looking for any government entities that are interested in using the latest AI / Deep Learning / ConvNets for use in space.

I assume there might be interest doing ML things at the point of sensor data collection, but I can't really locate anything.  Do you have any thoughts on this ?

I will be happy to cite you and your work, any quotes in my slide deck :)
